design/airgap-existing-cluster.md

```diff
@@ -0,0 +1,109 @@
+# Support for installing KOTS Admin Console in Airgapped Clusters
+
+## Goals
+
+- Install kotsadm to an existing cluster where the cluster does not have internet access
+- Install kotsadm to an existing cluster from a workstation where the workstation does not have internet access at the time of installation
+- Support different credentials for push / pull (workstation / cluster) to the local registry
+- Support temporary/dynamic and persistent/static credentials for pushing kotsadm images to the registry
+- Support only persistent/static credentials for pulling kotsadm images to the registry
```

Comment:
Are these credentials really needed?  All kotsadm images are public, but also they will be downloaded as a github asset according to this doc.

Reply:
should this say "pushing" to the registry? I think that's how I had read it

Reply:
The problem is just that the enterprise isn't going to have a registry that is open, so we expect they'll need a password to fetch the images, even if the images are otherwise public. But also just realized this is phrased wrong. The password isn't necessary to pull images **to** the registry, but **from** the local registry.

Reply:
```suggestion
- Support only persistent/static credentials for pulling kotsadm images from the airgapped registry
```